src/ChatListPage.js

```diff
@@ -0,0 +1,78 @@
+import React,{useState} from 'react';
+import { initialList } from './App';
+import styled from 'styled-components';
+import {IoIosText} from 'react-icons/io';
+import FriendState from './FriendState';
+import {Link} from 'react-router-dom';
+
+const ChatListPageContainer = styled.div`
+    margin-left: 70px;
+`;
+const Top = styled.div`
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+`;
+
+const H3 = styled.h3`
+    padding-left: 10px;
+`;
+const Icon = styled.div`
+    padding-right: 10px;
+`;
+
+const Input = styled.input`
+    width: 98%;
+    margin: 10px;
+    margin-left: 0px;
+    height: 30px;
+    border: solid 1px #dcdcdc;
+    border-radius: 10%;
+    display: block;
+`;
+
+const StyledLink = styled(Link)`
+    text-decoration: none;
+    color: black;
+`;
+
+
+const ChatListPage = () => {
+    const friendList = initialList;
+    const [text, setText] = useState('');
+
+    const onChangeText = (e) => {
+        setText(e.target.value);
+    }
+
+    return(
+        <ChatListPageContainer>
+            <Top>
+                <H3>채팅</H3>
+                <Icon>
+                    <IoIosText size="24" padding-right="10"/>
+                </Icon>
+            </Top>
+            <Input placeholder=" 채팅방, 참여자 검색" value={text} onChange={onChangeText} />
+            {friendList.map((v) => {
+                if (text === ''){
+                    return (
+                        <StyledLink to={`/ChattingPage/${v.id}`}>
+                            <FriendState props={v} where={'ChatList'}/>
+                        </StyledLink>
+                    );
+                } else {
+                    if (v.name.includes(text)){
```

Comment:
승범님도 includes를 통해서 검색 기능을 구현하셨네요! 저도 이렇게 했는데, 이 방법이 영어로 검색할 때는 대문자까지 그대로 검색해야만 되는 거라 원래의 메신저 검색 기능을 온전히 구현하지 못하는 것 같아요. 그래서 이런 식으로 바꿔주면 어떨까요? 저도 수정하려고 했는데 깜빡한 부분이라 아쉬움이 남아서 코멘트 남겨요.
```suggestion
                    if (v.name.toLowerCase().includes(text)){
```

Reply:
정말 세심한 포인트네요!